mautic/mautic
# Fix Kenya Regions

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | X
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description: Corrected Kenyan regions to reflect the correct and more appropriate counties

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Edit a contact, and in State, the options for Kenya are very limited.  See https://en.wikipedia.org/wiki/ISO_3166-2:KE for official list of Counties.

#### Steps to test this PR:
1. Confirm limited set of five options as above
2. Apply this PR
3. Clear cache
4. Edit contact, and you should see the full list of counties
